core/src/main/java/org/kiwix/kiwixmobile/core/utils/files/FileLogger.kt

```diff
@@ -31,26 +31,25 @@ import javax.inject.Singleton
 @Singleton
 class FileLogger @Inject constructor() {
 
-  fun writeLogFile(context: Context): File {
+  fun writeLogFile(context: Context, shouldWriteDeviceLogs: Boolean = true): File {
     // Create a new folder in private storage with name: logs
     val logDir = File(context.filesDir, "logs")
     val logFile = File(logDir, fileName)
 
-    Log.d("KIWIX", "Writing all logs into [" + logDir.path + "]")
+    Log.d(TAG, "Writing all logs into [" + logDir.path + "]")
 
-    if (!logDir.exists()) {
+    if (logDir.exists()) {
```

Comment:
Aren't we missing on the `logFile.isFile` check here?

Reply:
@Sagar0-0 Thanks for reviewing. Yes, of course, We have removed that check from here because we are deleting all the log files from this folder and then creating the new file on which we are saving the current logs. Since our `fileName`  contains the `currentMiliSeconds` in it, so two file names never be gonna same so that check is never gonna true.